src/Nordea/Components/FileDownload.elm

```diff
@@ -0,0 +1,143 @@
+module Nordea.Components.FileDownload exposing (..)
```

Comment:
I think it is a good idea to only explicitly expose what we need and not everything

Reply:
I agree. I will change it